README.md

```diff
@@ -25,6 +25,31 @@ This is the official Python client for interacting with our powerful [API](https
 
 ---
 
+
+
+## Table Of Contents
+
+**[1. Installation](#installation)** \
```

Comment:
Either of numbering/bullet point should be good. I'm not sure if we need both. Could we use just bullet points

Reply:
I have made the changes.Can you check.